doc/content/htgr/generic-pbr-tutorial/step1.md

```diff
@@ -0,0 +1,180 @@
+density and viscosity. Density and viscosity are named `rho` and `mu`.
+Note that `T_fluid` is set to a constant value in this input file because
+we do not solve the heat equation and therefore have no fluid temperature defined
+as a variable.
+
+This object also requires providing functors to the `porosity`, `speed`, and `characteristic_length`
+parameters.
+
+Porosity is defined as an auxiliary variable here:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=AuxVariables
+
+and set equal to the `bed_porosity` parameter of 0.39.
+`speed` is a functor that is computed automatically by the Navier-Stokes FV action,
+and `characteristic_length` is only used for computing Reynolds numbers, which are not
+required in Step 1; therefore, we set `characteristic_length = 1` for now.
+
+### Setting up the Thermal-Hydraulics Problem
+
+The thermal-hydraulics problem is set up using the finite volume Navier-Stokes action.
+This action is discussed in detail [here](https://mooseframework.inl.gov/source/actions/NSFVAction.html).
+The corresponding block in the input file is:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=Modules
+
+Understanding the finite volume Navier-Stokes action is important.
+Therefore, each line will be explained.
+
+!listing htgr/generic-pbr-tutorial/step1.i start=compressibility end=porous_medium_treatment
+
+This line indicates that weakly compressible fluid equations are solved. These equations allow the density of the fluid to vary, and the discretization methods assume Mach numbers of less than 0.2.
+For compressible flows in reactors, the weakly compressible formulation is the most commonly used equation set.
+
+!listing htgr/generic-pbr-tutorial/step1.i start=porous_medium_treatment end=density
+
+This line activates the porous flow treatment. Porous flow indicates that fluid
+and solid share a volume and exchange mass, momentum, and energy in this volume.
+This exchange is usually described by correlations. In this input file, no mass, momentum, or energy exchange occurs. The porosity defines how
+much of each porous flow element is fluid. The porosity in this input file is defined
+as a variable and provided in the line:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=NavierStokesFV start=porosity end=initial_velocity
+
+The names of the density and dynamic viscosity functors need to be provided to the
+action:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=NavierStokesFV start=density end=porosity
+
+Note that these functors are defined by the `GeneralFunctorFluidProps` object discussed above.
+
+Initial conditions for the velocity and pressure are defined by the lines:
+
+!listing htgr/generic-pbr-tutorial/step1.i block=NavierStokesFV start=initial_velocity end=inlet_boundaries
+
+Note that, since we solve a pseudo-transient, the initial conditions should be interpreted
+as initial guesses of the solution.
+We set the initial guess of the velocity to 0 and the initial guess of pressure to $5.4$ MPa.
```

Comment:
dont do 0 for velocity. This will take down the solver for advected quantities

Reply:
Changed to `initial_velocity = '1e-6 1e-6 0'`. But why does zero initial guess cause trouble for passive scalars?